modules/nlu/src/backend/predict-pipeline.ts

```diff
@@ -493,8 +505,9 @@ function MapStepToOutput(step: PredictStep, startTime: number): PredictOutput {
         confidence: s.confidence,
         name: s.name,
         source: s.source,
-        value: s.value
-      } as sdk.NLU.Slot
+        value: s.value,
+        entity: entitiesMapper(s.entity)
```

Comment:
if entity is null then entity is going to be undefined, is that what we want or we want it to be null  ?

Reply:
:/ I don't really care, but I think you're right. I'll simply return like this:

```
if (!e) {
      return e
    }
```